packages/connectivity/src/scp-cf/destination/destination-from-env.spec.ts

```diff
@@ -206,3 +207,27 @@ describe('env-destination-accessor', () => {
     });
   });
 });
+
+describe('registerDestination', () => {
+  const mockDestination: Destination = {
+    name: 'MockedDestination',
+    url: 'https://example.com'
+  };
+
+  afterEach(() => {
+    unmockDestinationsEnv();
+    jest.resetAllMocks();
+  });
+
+  it('should set the destination in the environment variables', () => {
```

Comment:
Add a test to see the errors are thrown for empty name destinations

Reply:
Also add a test for the `getDestination(name)` case to see that the destination by name is working.

Reply:
Also add a test to see the name conflict exception being raised.